phenol-annotations/src/main/java/org/monarchinitiative/phenol/annotations/io/hpo/Aspect.java

```diff
@@ -22,8 +45,18 @@ public static Optional<Aspect> parse(String aspect) {
         return Optional.of(Aspect.C);
       case "M":
         return Optional.of(Aspect.M);
+      case "H":
+        return Optional.of(Aspect.H);
       default:
         return Optional.empty();
     }
   }
+
+  public static Optional<Aspect> fromTermId(TermId termId) {
+    if (BY_TERMID.containsKey(termId.toString())){
```

Comment:
In Java, the `Map` will give you `null` if the item is not there. Therefore, the method can be simplified to 

```java
public static Optional<Aspect> fromTermId(TermId termId) {
    return Optional.ofNullable(BY_TERMID.get(termId.getValue()));
  }
```
This code should also be faster because only one `Map` lookup is performed.

So, your function looks more like Python, where getting the value raises a `KeyError` so, we have to either check if the key is in the `dict` or catch the `KeyError`.